src/Microsoft.DncEng.PatGeneratorTool/Program.cs

```diff
@@ -130,7 +132,16 @@ private static async Task<int> Go(List<AzureDevOpsPATScopes> scopes, string[] or
                     user = $"{user.Split('\\').Last()}@microsoft.com";
                 }
 
-                credentials = new VssAadCredential(user, password);
+                var app = PublicClientApplicationBuilder
+                    .Create(AzdoClientId)
+                    .WithAuthority("https://login.microsoftonline.com/microsoft.com/")
+                    .Build();
+
+                var passwordSecureStr = new SecureString();
```

Comment:
Do these things really require a "SecureString"?  I thought .NET bailed on those a long time ago... Part of our dogfood feedback should be "don't do that".

Reply:
It does for some reason. Their API changed to require it. https://github.com/dotnet/platform-compat/blob/master/docs/DE0001.md says we should no longer use it.

Reply:
I'm going to add you to the dogfood chat for the libraries. You can report issues to them.

Reply:
> I'm going to add you to the dogfood chat for the libraries. You can report issues to them.

Do they own `Microsoft.Identity.Client`? The SecureString usage is there not in the DevOps library.

Reply:
Ahh, nope. I'd report issues in the libraries themselves, or in transitive dependencies that the library introduces. For instance, if their library _requires_ use of another API that requires `SecureString`, that would be worth mentioning. They may be able to find other options.

Reply:
There is a "survey" for that library (the MSAL stuff). I filled it out saying it shouldn't use SecureString.  Feel free to do the same. :-)